lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/interfaces.hpp

```diff
@@ -111,4 +96,62 @@ class VLH35_Angle_Bus {
   static void select_enc_(uint8_t id);
 };
 
+class KillSwitchRelay {
+public:
+  static bool dead;
+  static long kill_time;
+
+  static void init();
+  
+  //these two functions deal with the relay that kills all power to the motors
+  static void reset();
+  static void kill();
+
+  //the main loop, to be run before the effort values are assigned to motors
+  static void logic(RobotEffort &effort);
+
+  //sets an individual motor to 0% power.
+  static void kill_motor(int id, RobotEffort &effort);
```

Comment:
from a wording perspective. i think the difference between kill_motor() and kill() should be different. change kill_motor to disable_motor and keep kill() for the ultimatum safety action

Reply:
👍